tests/e2e/hcp_cluster_test.go

```diff
@@ -182,4 +184,69 @@ var _ = Describe("HCP cluster testing",
 					Should(ContainSubstring(
 						"ERR: Listing identity providers is not supported for clusters with external authentication configured"))
 			})
+
+		It("can edit ROSA HCP cluster with additional allowed principals - [id:74556]",
+			labels.High, labels.Runtime.Day2,
+			func() {
+				By("Check the help message of 'rosa edit cluster -h'")
+				helpOutput, err := clusterService.EditCluster("", "-h")
+				Expect(err).To(BeNil())
+				Expect(helpOutput.String()).To(ContainSubstring("--additional-allowed-principals"))
+
+				By("Check if cluster profile is enabled with additional allowed principals")
+				if clusterConfig.AdditionalPrincipals == "" {
```

Comment:
I believe there should be a scenario that set the `--additional-allowed-principles` as day2